Dockerfile

```diff
@@ -16,21 +16,23 @@ ENV LOUNGE_SRC "${LOUNGE_HOME}/src"
 RUN groupadd --gid ${gid} ${group} \
       && useradd --home "${LOUNGE_HOME}" --create-home --uid ${uid} --gid ${gid} ${user}
 
-VOLUME "${LOUNGE_DATA}"
-RUN chown -R ${user} "${LOUNGE_DATA}"
-
-# Drop root.
-USER ${user}
-
 # Install thelounge.
 RUN mkdir -p "${LOUNGE_SRC}"
 WORKDIR "${LOUNGE_SRC}"
 RUN curl -L $(npm view thelounge@${LOUNGE_VERSION} dist.tarball) | tar --strip-components 1 -xzf -
 RUN npm install
+RUN npm link
```

Comment:
I feel like this is becoming really bizarre. Why not just use `npm install -g`? `npm link` is usually used to link ongoing projects so they can be tested while they are being accessible globally. For example [from the doc](https://docs.npmjs.com/cli/link):

> This is handy for installing your own stuff, so that you can work on it and test it iteratively without having to continually rebuild.